lib/rufus/lua/state.rb

```diff
@@ -10,10 +10,7 @@ module Rufus::Lua
   # The entry point of rufus-lua is Rufus::Lua::State, look there.
   #
   module StateMixin
-
-    LUA_GLOBALSINDEX = -10002
-    LUA_ENVIRONINDEX = -10001
```

Comment:
Global and environindex were removed in Lua 5.2:
https://www.lua.org/manual/5.2/manual.html#8

>Pseudoindex LUA_GLOBALSINDEX was removed. You must get the global environment from the registry (see [§4.5](https://www.lua.org/manual/5.2/manual.html#4.5)).
Pseudoindex LUA_ENVIRONINDEX and functions lua_getfenv/lua_setfenv were removed, as C functions no longer have environments.